Jenkinsfile

```diff
@@ -1,8 +1,8 @@
 pipeline {
   agent {
     kubernetes {
-      label 'xtext-xtend-api-diff-' + env.BUILD_NUMBER
-      defaultContainer 'xtext-buildenv'
+      label 'xtext-xtend-api-diff'
```

Comment:
Why don't we remove the custom pod completely? I don't see anything special in this podtemplate, maybe we shouln't maintain it at all then. (See https://github.com/eclipse/xtext-core/blob/master/Jenkinsfile for example)

Reply:
without it seems slow like crap

Reply:
The do offer additional labels for more resources, in xtext-xtend we use:
> label 'centos-7-6gb'

I did a quick search, but couldn't find a list of available labels, but it might be worth it to try this one...

Reply:
its working anyway so i think we are fine. maybe just a slow machine in pool problem